{{cookiecutter.project_slug}}/_/deployment/python/deployment/application/base/cronjob.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: cron-backup
+spec:
+  schedule: "1 1 * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: application
+            image: IMAGE
+{%- if cookiecutter.framework == 'Django' %}
+            command: ['/bin/bash', '-c']
+            args: 
+              - python manage.py dumpdata -o backups/db.json ;
+              - cat backups/db.json
```

Comment:
I'm not sure if this will work. Wouldn't that result in something like this?
```console
/bin/bash -c "python manage.py dumpdata -o backups/db.json ;" "cat backups/db.json"
```

Reply:
That's what I expected as well, but this solution seems to work.  This solution is what I found when searching online for how to run multiple commands from the yaml file (after several other ideas didn't work).  If you know of a cleaner way to do it, I'd be happy to change this.

Reply:
Alright, if it works, fine with me. On the other hand, again, the `cat` command produces output, which may signal a "failing job" IIRC (see other, related comments).

Reply:
I think it's ok for it to give error messages to prompt the user to replace it with a real job.